File: 백준/Silver/1978. 소수 찾기/소수 찾기.java
import java.util.*;
public class Main {
 
	public static void main(String[] args) {
		Scanner sc =new Scanner(System.in);
		int N = sc.nextInt();
		int count =0;
		for(int i=0;i<N;i++) {
			if(isPrime(sc.nextInt()))
				count++;
		}
		System.out.println(count);
		
		
	}
	public static boolean isPrime(int k) {
		if(k==1) return false;
		for(int i =2;i<=Math.sqrt(k);i++)
			if(k%i==0) {
				return false;

			}
		return true;
	}
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
import math
def isPrime(num):
    if num == 1:
        return False
    for i in range(2,num):
        if num%i == 0:
            return False
        if( i > math.sqrt(num)): break

    return True

n = int(input())
numList = list(map(int,input().split(" ")))
count =0
for i in numList:
    if isPrime(i):
        count+=1

print(count)
